### DISCUSSION TITLE
I have this error when try to call the api using node

### DISCUSSION THREAD

**Discussion by Ziputrafa** [deepgram/community#550] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

```
`Error: Sorry, `listenFile` does not appear to be a valid operation on this API.
    at Proxy.<anonymous>`
```

and I'm writing this on node. I'm using Multer to get the file from the client.

```js

const express = require('express');
const cors = require('cors');
const multer = require('multer');

const sdk = require('api')('@api-beta-deepgram/v1.0#4xe1d1llqyfzn4x');

const upload = multer({ dest: 'uploads/' });
const app = express();

app.use(cors());
app.use(express.json());

const PORT = 8000;

app.post('/api', upload.single('video'), (req, res) => {
  sdk.auth('Token *******');
  sdk
    .listenFile(req.file.path)
    .then(({ data }) => console.log(data))
    .catch((err) => console.error(err));
});

app.listen(PORT, () => {
  console.log(`running on port ${PORT}`);
});
`
```

Is there something that I'm missing here? and also my question is `sdk.listenFile('the path of the file')`. 

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

'@api-beta-deepgram/v1.0#4xe1d1llqyfzn4x'

### If possible, please attach an example audio file to reproduce the issue.

Uploading video.mp4…

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@Ziputrafa  What Deepgram SDK version are you trying to use here? I see this line but not quite sure what it is you are attempting to do.

```const sdk = require('api')('@api-beta-deepgram/v1.0#4xe1d1llqyfzn4x');```

I'd suggest using our new [JS SDK version ](https://github.com/deepgram/deepgram-js-sdk) if you can.
